onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -14,6 +14,9 @@
 import math
 from typing import Any, Optional, Sequence, Tuple, Union
 
+import numpy as np
+import onnx.numpy_helper as onh
```

Comment:
```suggestion
```

onnx helpers tend to be protobuf-y and slow. Consider using ir.tensor, ir.DataType and related methods instead"

Reply:
In that case, it just a tensor with one element. Can you tell how to replace by ir.Value then? I think I should use ir.tensor... The documentation should include an example where you build an onnx model from scrapt with the IR. That would be very helpful.

Reply:
> documentation 

I agree. Need to find time to work on that

> how to replace

Commented below. LMK if you need more info, thanks!